spec/clean-spec.js

```diff
@@ -1,98 +1,74 @@
+    app.get('/tarball/test-module-1.2.0.tgz', (_request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'test-module-1.2.0.tgz')));
     server = http.createServer(app);
-
     let live = false;
-    server.listen(3000, '127.0.0.1', function() {
-      console.log("Server started");
-      const atomHome = temp.mkdirSync('apm-home-dir-');
-      process.env.ATOM_HOME = atomHome;
-      process.env.ATOM_ELECTRON_URL = "http://localhost:3000/node";
+    server.listen(3000, '127.0.0.1', () => {
+      console.log('Server started');
+      process.env.ATOM_HOME = temp.mkdirSync('apm-home-dir-');
+      process.env.ATOM_ELECTRON_URL = 'http://localhost:3000/node';
       process.env.ATOM_ELECTRON_VERSION = nodeVersion;
       process.env.npm_config_registry = 'http://localhost:3000/';
-
       moduleDirectory = path.join(temp.mkdirSync('apm-test-module-'), 'test-module-with-dependencies');
       wrench.copyDirSyncRecursive(path.join(__dirname, 'fixtures', 'test-module-with-dependencies'), moduleDirectory);
       process.chdir(moduleDirectory);
-      return live = true;
+      live = true;
     });
-    return waitsFor(() => live);
-  });
+    waitsFor(() => live);
+  })
 
-  afterEach(function() {
-    let done = false;
-    server.close(() => done = true);
-    return waitsFor(() => done);
+  afterEach(() => {
+    let done = false
+    server.close(() => {
+      done = true;
+    });
+    waitsFor(() => done);
   });
 
-  it('uninstalls any packages not referenced in the package.json', function() {
+  it('uninstalls any packages not referenced in the package.json', () => {
     const removedPath = path.join(moduleDirectory, 'node_modules', 'will-be-removed');
     fs.makeTreeSync(removedPath);
-    fs.writeFileSync(
-      path.join(removedPath, 'package.json'),
-      '{"name": "will-be-removed", "version": "1.0.0", "dependencies": {}}',
-      'utf8'
-    );
-
+    fs.writeFileSync(path.join(removedPath, 'package.json'), '{"name": "will-be-removed", "version": "1.0.0", "dependencies": {}}', 'utf8');
     const callback = jasmine.createSpy('callback');
     apm.run(['clean'], callback);
-
     waitsFor('waiting for command to complete', () => callback.callCount > 0);
-
-    return runs(function() {
+    runs(() => {
       expect(callback.mostRecentCall.args[0]).toBeUndefined();
-      return expect(fs.existsSync(removedPath)).toBeFalsy();
+      expect(fs.existsSync(removedPath)).toBeFalsy();
     });
   });
 
-  return it('uninstalls a scoped package', function() {
+  it('uninstalls a scoped package', () => {
     const removedPath = path.join(moduleDirectory, 'node_modules/@types/atom');
     fs.makeTreeSync(removedPath);
-    fs.writeFileSync(
-      path.join(removedPath, 'package.json'),
-      '{"name": "@types/atom", "version": "1.0.0", "dependencies": {}}',
-      'utf8'
-    );
-
+    fs.writeFileSync(path.join(removedPath, 'package.json'), '{"name": "@types/atom", "version": "1.0.0", "dependencies": {}}', 'utf8');
```

Comment:
If undoing the above indentation change, please do this one as well. (If leaving them, that is also okay.)